pynbody/plot/sph.py

```diff
@@ -375,14 +379,14 @@ def image(sim, qty='rho', width="10 kpc", resolution=500, units=None, log=True,
         is_projected = _units_imply_projection(sim, qty, units)
 
     if is_projected:
-        kernel = sph.Kernel2D()
+        kernel = sph.Kernel2D(type=kernel_type)
```

Comment:
This line should read `kernel = sph.Kernel2D(kernel)`, to construct a 2D kernel over whatever kernel has been previously selected